### PR TITLE
Feature/l15-network-reset

### PR DESCRIPTION
**Pull request by rryter**
Ready for review, but do not merge yet.

### CODE REVIEW

**Comment by rryter** [comment]
Screenshot 2021-11-02 at 08 35 59
Screenshot 2021-11-02 at 08 36 09

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.4",
-    "@docusaurus/preset-classic": "2.0.0-beta.4",
+    "@docusaurus/core": "^2.0.0-beta.8",
+    "@docusaurus/preset-classic": "^2.0.0-beta.8",
```
Comment (Hugoo) [inline package.json:18]: Jean also updated this in his PR:

https://github.com/lukso-network/docs/pull/33

Last one to merge may have conflicts 😁😅

**Review by Hugoo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rryter) [reply package.json:18]: 👍

Comment (Hugoo) [reply package.json:18]: And Reto, you are the merge conflict winner haha 🙈

Comment (rryter) [reply package.json]: image
😄

**Review by mxmar — APPROVED** [review]

**Pull request merged**